### DISCUSSION TITLE
How can it have multiple sheet support when `sc-im` only uses `.csv` which is made of one sheet documents?

### DISCUSSION THREAD

**Discussion by freddieventura** [andmarti1424/sc-im#661] [General]
Hi, I didn't want to post thisone on the Issues section, I may be a bit hard to understand this, but can someone explain me this? I'm not able so save a multiple sheet document as it will take the last edited sheet as the one to use , therefore deleting the previousone.

Does that has to be with the fact (I'm not 100% sure) that `csv` format only supports one sheet files?

Thank you

**Comment by 1dancook** [comment]
Hi there, I'm not 100% clear on what you are trying to do. Can you write out the steps in order?

sc-im uses its own format and that format supports a great number of things that are impossible in csv. Try `sc-im test.sc` add a number of things, save, and inspect the file in a text editor to see.

Exporting to CSV is possible. My assumption (haven't tried and can't check atm) is that exporting into CSV would use the current sheet if there are multiple sheets. CSV does not support multiple sheets - basically CSV is just one table of rows and delimited columns.

**Reply by freddieventura** [reply]
Yeah well , my issue was that I am managing a shared folder with `Google Drive` , and locally I'm using `sc-im` , so I'm limited to its formats. Either `.csv` or `.sc` are not supported by `Google Sheets` so I needed to create a duplicate of them in `xlsx` via `unoconv -f xlsx ./*.csv` , so I do that recursively when exporting and importing the other way around .
```
find /myPath/ -type f -iname '*.csv' -exec unoconv -f xlsx {} \;
``` 

That is the only workaround I can find at the moment.

**Reply by 1dancook** [reply]
I think you can export from sc-im to xlsx and multiple sheets works. I just tried it and it's okay.

I think in another comment somewhere you had some problem with getting xlsx support to work so hopefully you can get that working.

I don't use xlsx for anything at the moment so haven't tried the functionality in sc-im. Im afraid I can't provide much other feedback.


---

@andmarti1424 one thing I just noticed is the sheet name didn't make its way into the xlsx.

```
Put some values in cells
:newsheet "hi"
Put some values in cells
:e xlsx test.xlsx
Open sheet -> has "sheet 1" and "sheet 2" and no "hi" sheet.
```
